package.json

```diff
@@ -45,7 +45,7 @@
     "localforage": "1.7.1",
     "ngeo": "https://api.github.com/repos/camptocamp/ngeo/tarball/f3e26bf",
     "nomnom": "1.8.1",
-    "ol-cesium": "1.34.0",
+    "ol-cesium": "https://api.github.com/repos/openlayers/ol-cesium/tarball/9bc93a85a2cf25fbec11905597f3c5cbc18acea5",
```

Comment:
Why not take v1.37.0?

Reply:
I've changed this line in a separate commit, because that was a conflict resolution error I've made. This line come simply from the master.